packages/demo/usecase-demo/src/routes/_explore/content.$protectedDataAddress.tsx

```diff
@@ -30,6 +31,10 @@ export function ProtectedDataPreview() {
     address: protectedDataAddress,
   });
 
+  const router = useRouter();
+  const hasPrevionPage = router.history.location.state.key ? true : false;
```

Comment:
Typo to be fixed:

`hasPrevionPage` → `hasPreviousPage`

Reply:
Ternary is probably unnecessary here, we can simplify:

```
const hasPreviousPage = !!router.history.location.state.key;
```

Reply:
done